kctf-operator/pkg/controller/challenge/challenge_default.go

```diff
@@ -16,8 +16,7 @@ func PersistentVolumeClaimsDefault(challenge *kctfv1alpha1.Challenge) *corev1.Pe
 		Items: []corev1.PersistentVolumeClaim{
```

Comment:
Do I understand it correctly that this adds a persistent volume to every challenge by default?

Reply:
Yes, but I'm changing that